### PR TITLE
SAK-42814 Signup title is not initialized

### PR DESCRIPTION
**Pull request by ern**
Entire class lombokified

### CODE REVIEW

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 7
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/sakaiproject/sakai/pullRequest?prid=4633341&bid=15528148)

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 7
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/sakaiproject/sakai/pullRequest?prid=4633341&bid=15528148)

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/sakaiproject/sakai/pullRequest?prid=4633341&bid=15528148)

File: signup/tool/src/java/org/sakaiproject/signup/tool/jsf/NewSignupMeetingBean.java [hunk 1]
```diff
@@ -59,2204 +61,1285 @@
...
+    public static final boolean DEFAULT_ALLOW_COMMENT = "true".equalsIgnoreCase(Utilities.getSignupConfigParamVal("signup.default.allow.comment", "true"));
+    public static final boolean DEFAULT_AUTO_RIMINDER = "true".equalsIgnoreCase(Utilities.getSignupConfigParamVal("signup.event.default.auto.reminder", "true"));
+    public static final boolean DEFAULT_AUTO_RMINDER_OPTION_CHOICE = "true".equalsIgnoreCase(Utilities.getSignupConfigParamVal("signup.autoRiminder.option.choice.setting", "true"));
+    public static final boolean DEFAULT_USERID_INPUT_MODE_OPTION_CHOICE = "true".equalsIgnoreCase(Utilities.getSignupConfigParamVal("signup.userId.inputMode.choiceOption.setting", "true"));
+    public static final boolean DEFAULT_EXPORT_TO_CALENDAR_TOOL = "true".equalsIgnoreCase(Utilities.getSignupConfigParamVal("signup.default.export.to.calendar.setting", "true"));
+    public static final boolean DEFAULT_CREATE_GROUPS = "true".equalsIgnoreCase(Utilities.getSignupConfigParamVal("signup.default.create.groups.setting", "true"));
+    public static final String DEFAULT_SEND_EMAIL_TO_SELECTED_PEOPLE_ONLY = Utilities.getSignup
```
Comment (mpellicer) [inline signup/tool/src/java/org/sakaiproject/signup/tool/jsf/NewSignupMeetingBean.java]: We need a default here.

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/sakaiproject/sakai/pullRequest?prid=4633341&bid=15528148)

**Pull request merged**